### ISSUE TITLE
Fixed Tax Rate Not Working!

### ISSUE THREAD

**Issue by erdalp** [smartstore/Smartstore#665]
Hello,

After changing the server in my project, the rate information does not appear in the Fixed Tax Rate section. When I try to update it does not update. But there are related records in the sql database. I see the relevant records in both the TaxCategory table and the setting table. The interesting thing is that when I connect to the same database locally and run the project, it works without any problems.

https://user-images.githubusercontent.com/20066079/233952588-84860f87-c483-4a95-9f00-d4372e14d8b0.mp4

![image](https://user-images.githubusercontent.com/20066079/233959309-2c53e1a2-5d54-4113-b384-e7ee41d44215.png)

**Comment by Michael-Herzog** [comment]
This problem also can't be reproduced. Please check you logs for any hints regarding this problem.

**Comment by erdalp** [comment]
Unfortunately, there is no log entry generated for this. There is no record in either the project's logs or the Windows logs.

**Comment by erdalp** [comment]
Thank you very much for your support. I solved the problem.

I followed a way like this. First of all, I thought that the problem might be due to antivirus restrictions on the server, so I uninstalled the antivirus.

Later, when I debugged the project, I noticed an error like this; Could not retrieve data with ICacheManager. When I examined this in more detail, it made me think that it was due to the fact that I moved the database to the new server in the form of Scripts and did not receive the indexes.

Therefore, when I did the operation again, the error was resolved.
